src/core/Akka.Tests/Actor/Scheduler/SchedulerShutdownSpec.cs

```diff
@@ -0,0 +1,137 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="SchedulerShutdownSpec.cs" company="Akka.NET Project">
+//     Copyright (C) 2009-2016 Lightbend Inc. <http://www.lightbend.com>
+//     Copyright (C) 2013-2016 Akka.NET project <https://github.com/akkadotnet/akka.net>
+// </copyright>
+//-----------------------------------------------------------------------
+
+using System;
+using System.Threading;
+using Akka.Actor;
+using Akka.Configuration;
+using Akka.Event;
+using Akka.TestKit;
+using Akka.Util.Internal;
+using FluentAssertions;
+using Xunit;
+
+namespace Akka.Tests.Actor.Scheduler
+{
+    /// <summary>
+    /// Need to guarantee that the <see cref="IScheduler"/>, if it implements <see cref="IDisposable"/>, is shutdown.
+    /// </summary>
+    public class SchedulerShutdownSpec
```

Comment:
Specs for verifying that:

1. It's possible to have an `IScheduler` automatically shut down via `IDisposable`.
2. That the default `IScheduler` does this.
3. That the default `IScheduler` throws a `SchedulerException` in the event of attempting to schedule after the scheduler has been shutdown.